resources/views/edit.blade.php

```diff
@@ -0,0 +1,32 @@
+@extends('layouts.master')
+@section('title','post')
+@section('content')
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>投稿</title>
+    <link href="node_modules\bootstrap\dist\css\bootstrap.min.css" type="text/css" rel="stylesheet" >
```

Comment:
```suggestion
    <link href="{{ asset('node_modules\bootstrap\dist\css\bootstrap.min.css' }}" type="text/css" rel="stylesheet" >
```
dùng hàm asset hoặc mix để lấy link ra nha